chart/templates/mayastor/obs/stats-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.eventing.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-event-store
+  labels:
+    app: event-store
+    {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+    {{ include "label_prefix" . }}/version: {{ .Chart.Version }}
+data:
```

Comment:
Is this going to be populated by the components?

Reply:
The data field is going to be populated from code . If we have the value here then we need to update when the schema changes. So no data here .